### ISSUE TITLE
New row in the table with folders

### ISSUE THREAD

**Issue by alexneventchin** [sjarvela/kloudspeaker#158]
Hi , it's me again.Yesterday i noticed that file size row isn't working when its folder line ,not a file.So it will be nice if you can make "file size" to "Number of files" and add how many files are in the folder.
Last edited/uploaded also isn't working with folders.

Have a nice day :)

**Comment by sjarvela** [comment]
That's right, because file size is something that the OS itself reports for files. Folders don't have same kind of info, it would have to be calculated/resolved every time, going through all subfolders and files.

While it is certainly possible, it can be heavy. If you want to display this info in the list view, it would mean that _every time_ some user views or refreshes the list, it must traverse entire file tree.

Caching would help, but since there is no hooks etc to know if filesystem has changed, there is no fool proof way to know when to invalidate the cache.
